### PR TITLE
[Validator] review latvian translations

### PR DESCRIPTION
**Pull request by k0d3r1s**
| Q             | A
| ------------- | ---
| Branch?       | 5.4
| Bug fix?      | yes
| New feature?  | no
| Deprecations? | no
| License       | MIT

Translated new keys (114, 115) into latvian

### CODE REVIEW

File: src/Symfony/Component/Validator/Resources/translations/validators.lv.xlf [hunk 1]
```diff
@@ -444,11 +444,11 @@
             </trans-unit>
             <trans-unit id="114">
                 <source>This value is too short. It should contain at least one word.|This value is too short. It should contain at least {{ min }} words.</source>
-                <target state="needs-translation">This value is too short. It should contain at least one word.|This value is too short. It should contain at least {{ min }} words.</target>
+                <target>Šī vērtība ir pārāk īsa. Tai būtu jābūt vismaz vienu vārdu garai.|Šī vērtība ir pārāk īsa. Tai būtu jābūt ne mazāk kā {{ limit }} vārdus garai.</target>
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Validator/Resources/translations/validators.lv.xlf]:
```suggestion
                <target>Šī vērtība ir pārāk īsa. Tai būtu jābūt vismaz vienam vārdam.|Šī vērtība ir pārāk īsa. Tai būtu jābūt ne mazāk kā {{ min }} vārdiem.</target>
```

File: src/Symfony/Component/Validator/Resources/translations/validators.lv.xlf [hunk 2]
```diff
@@ -444,11 +444,11 @@
...
             </trans-unit>
             <trans-unit id="115">
                 <source>This value is too long. It should contain one word.|This value is too long. It should contain {{ max }} words or less.</source>
-                <target state="needs-translation">This value is too long. It should contain one word.|This value is too long. It should contain {{ max }} words or less.</target>
+                <target>Šī vērtība ir pārāk gara. Tai būtu jābūt vienam vārdam.|Šī vērtība ir pārāk gara. Tai būtu jābūt ne vairāk kā {{ limit }} vārdus garai.</target>
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Validator/Resources/translations/validators.lv.xlf]:
```suggestion
                <target>Šī vērtība ir pārāk gara. Tai būtu jābūt vienam vārdam.|Šī vērtība ir pārāk gara. Tai būtu jābūt ne vairāk kā {{ max }} vārdiem.</target>
```

**Review by nicolas-grekas — COMMENTED** [review]
WDYT of these suggestions?

**Comment by javiereguiluz** [comment]
Ping @psihius who is native in Latvian too. Thanks!

File: src/Symfony/Component/Validator/Resources/translations/validators.lv.xlf (hunk 1 above)
Comment (k0d3r1s) [reply src/Symfony/Component/Validator/Resources/translations/validators.lv.xlf]: @nicolas-grekas both works but i chose this as it is more consequent with other translations

File: src/Symfony/Component/Validator/Resources/translations/validators.lv.xlf (hunk 2 above)
Comment (k0d3r1s) [reply src/Symfony/Component/Validator/Resources/translations/validators.lv.xlf]: @nicolas-grekas same reason as other for other comment

File: src/Symfony/Component/Validator/Resources/translations/validators.lv.xlf (hunk 1 above)
Comment (nicolas-grekas) [reply src/Symfony/Component/Validator/Resources/translations/validators.lv.xlf]: ok, then there's still the placeholder: limit doesn't exist, it's either max or min

Comment (k0d3r1s) [reply src/Symfony/Component/Validator/Resources/translations/validators.lv.xlf]: oops, typo from other translation. fixed

**Review by nicolas-grekas — APPROVED** [review]

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @k0d3r1s.

**Pull request merged**